core21/Informix.DB2/Tests/Linq/GroupByTests/Tests.Linq.GroupByTests.GroupByCustomEntity1(Informix.DB2).sql

```diff
@@ -12,5 +12,6 @@ FROM
 			Child c_1
 	) t1
 GROUP BY
-	t1.ParentID
```

Comment:
Unrelated to issue, but it needs our attention at some point - sql optimizer didn't optimized this query for informix  as it was done for other providers

Reply:
Which could probably be a cause of several group by tests not working with informix